src/components/CheckboxInput/CheckboxInput.scss

```
@@ -0,0 +1,18 @@
+.checkboxInput {
+  display: flex;
+  align-items: center;
+
+  &.isDisabled {
```

Would it be better to use the `:disabled` pseudo class instead of creating a named nested css class?
```suggestion
  &:disabled {
```


https://developer.mozilla.org/en-US/docs/Web/CSS/:disabled

---

Not in this case. Here we're applying this class to a div, since it can't technically be disabled. The class is used to gray out the text, and disable pointer events, so, even if there is a link, it can't be clicked.

The actual input does have the `disabled` attribute applied and it handles disabling itself.